test/libraries/SafeMigration.spec.ts

```diff
@@ -54,9 +62,11 @@ describe("SafeMigration Library", () => {
 
     describe("constructor", () => {
         const setupTests = deployments.createFixture(async () => {
+            await deployments.fixture();
+
             return {
-                singletonAddress: await getSafeSingletonContract(),
```

Comment:
this function would return the singleton based on the environment variable, and we do not want this here - we want the "L1" singleton always